pkg/utils/scheduling/scheduling.go

```diff
@@ -60,6 +60,13 @@ func IsSchedulable(pod *v1.PodSpec, node *v1.Node) bool {
 	return true
 }
 
+func IsForcefullyEvicted(pod *v1.Pod) bool {
+	if pod.Status.Phase == "Failed" && pod.Status.Reason == "Evicted" {
```

Comment:
Do we need to specify evicted? What about just failed?

Reply:
Failed makes more sense here. Good point. [Failed containers phase](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle)
> All containers in the Pod have terminated, and at least one container has terminated in failure. That is, the container either exited with non-zero status or was terminated by the system.

Reply:
Since the pod won't be coming back, it makes sense to not count any pod that fails here.